collectors/feed/config.yml

```diff
@@ -0,0 +1,89 @@
+---
+#
+# This file includes a list of RSS feeds from which we periodically pull
+# the latest entries. This file uses the YAML syntax.
+#
+# Each entry *must* include a URL (field name "url") from which the feed
+# entries are pulled, and if available a components list (field name
+# "components" containing a list of related components).
+#
+# Each entry *must* include a "meta" section with a "description" and
+# "reason" fields, the purpose of which is:
+#
+# - description: includes short information about the feed we are scraping.
+#
+# - reason: includes reasoning for why we are watching a particular feed.
+#
+# IMPORTANT: Do not modify existing feed names (e.g. "name: Ruby on Rails
+# Security"). These are used to identify snippets and perform special
+# actions on certain types of feed snippets.
+
+## - url: https://www.us-cert.gov/ncas/all.xml
+##   name: US-CERT Advisories
+##   source: internet
+##   meta:
+##   - description: "The US-CERT Advisories feed includes vulnerability
+##       reports of the most frequent, high-impact security incidents
+##       currently being reported to the US-CERT."
+##   - reason: "This feed is watched mainly for redundancy to ensure we caught
+##       all security issues reported elsewhere. US-CERT rarely published an
+##       original advisories."
```

Comment:
Do we want to remove this source?

Reply:
It seems it's defunct, so most probably yes.

Reply:
It seems this has been folded into CISA, which AFAIK we already process elsewhere. I consider this done.